ring/http.go

```diff
@@ -26,21 +28,26 @@ const tpl = `
 						<th>State</th>
 						<th>Address</th>
 						<th>Last Heartbeat</th>
+						<th>Tokens</th>
+						<th>Ownership</th>
 						<th>Actions</th>
 					</tr>
 				</thead>
 				<tbody>
-					{{ range $key, $value := .Ring.Ingesters }}
+					{{ range .Ingesters }}
 					<tr>
-						<td>{{ $key }}</td>
-						<td>{{ $value.State }}</td>
-						<td>{{ $value.Hostname }}</td>
-						<td>{{ $value.Timestamp | time }}</td>
-						<td><button name="forget" value="{{ $key }}" type="submit">Forget</button></td>
+						<td>{{ .ID }}</td>
+						<td>{{ .State }}</td>
+						<td>{{ .Hostname }}</td>
+						<td>{{ .Timestamp }}</td>
+						<td>{{ .Tokens }}</td>
+						<td>{{ .Ownership }}%</td>
+						<td><button name="forget" value="{{ .ID }}" type="submit">Forget</button></td>
 					</tr>
 					{{ end }}
 				</tbody>
 			</table>
+			<pre>{{ .Ring }}</pre>
```

Comment:
This is going to render in a way that's not so helpful to human beings. We should probably pretty print it instead, w/ `MarshalIndent`.